### PR TITLE
Updated features list based on 0.8 release

### PR DESCRIPTION
**Pull request by jasmine-jaksic**
Updated features page on istio.io to reflect some additional changes from 0.8 release

### CODE REVIEW

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit 22fb5da6bb25b8e2d958b96952078782144cfc2b

https://app.netlify.com/sites/preliminary-istio/deploys/5b187c5fdd6a545236889f8e

File: content/about/feature-stages.md [hunk 1]
```diff
@@ -70,33 +75,34 @@ Below is our list of existing features and their current phases. This informatio
...
 | [Pilot Integration into Eureka](/docs/setup/consul/quick-start/) | Alpha
 | [Pilot Integration into Cloud Foundry Service Discovery](/docs/setup/consul/quick-start/)    | Alpha
 | [Basic Config Resource Validation](https://github.com/istio/istio/issues/1894) | Alpha
 | Mixer Telemetry Collection (Tracing, Logging, Monitoring) | Alpha
 | [Custom Mixer Build Model](https://github.com/istio/istio/wiki/Mixer-Adapter-Dev-Guide) | Alpha
 | Enable API attributes using an IDL | Alpha
-| [Helm](/docs/setup/kubernetes/helm-install/) | Alpha
+| [Helm](/docs/setup/kubernetes/helm-install/) | Beta
 | [Multicluster Mesh](/docs/setup/kubernetes/multicluster-install/) | Alpha
+| Custom Envoy Filters | Alpha
```
Comment (kyessenov) [inline content/about/feature-stages.md]: This was removed. New API design is here https://github.com/istio/api/pull/447, web hooks are gone in 0.8.

File: content/about/feature-stages.md [hunk 2]
```diff
@@ -70,33 +75,34 @@ Below is our list of existing features and their current phases. This informatio
...
+
 
 ### Core
 
 | Feature           | Phase
 |-------------------|-------------------
 | [Kubernetes: Envoy Installation and Traffic Interception](/docs/setup/kubernetes/)        | Beta
-| [Kubernetes: Istio Control Plane Installation](/docs/setup/kubernetes/) | Beta
+| [Kubernetes: Istio Control Plane Installation](/docs/setup/kubernetes/) | Alpha
+| [Kubernetes: Istio Control Plane Upgrade](/docs/setup/kubernetes/) | Beta
```
Comment (kyessenov) [inline content/about/feature-stages.md]: Why is upgrade beta but installation alpha? Seems like installation should be beta then?

File: content/about/feature-stages.md [hunk 3]
```diff
@@ -35,22 +35,26 @@ Below is our list of existing features and their current phases. This informatio
 | [Protocols: TCP](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/tcp_proxy)    | Alpha
 | [Protocols: WebSocket](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/websocket)      | Alpha
```
Comment (ldemailly) [inline content/about/feature-stages.md]: This doesn't exist/work anymore in 0.8 alpha3, change it to Not Started maybe?

File: content/about/feature-stages.md [hunk 4]
```diff
@@ -35,22 +35,26 @@ Below is our list of existing features and their current phases. This informatio
...
-| [Routing Rules: Header Rewrite](/docs/tasks/traffic-management/request-routing/)      | Alpha
-| [Routing Rules: Traffic Splitting](/docs/tasks/traffic-management/request-routing/)      | Alpha
+| [Request Routing](https://istio.io/docs/tasks/traffic-management/request-routing/)      | Alpha
+| [Fault Injection](https://istio.io/docs/tasks/traffic-management/fault-injection/)      | Alpha
+| [Traffic Shifting](https://istio.io/docs/tasks/traffic-management/traffic-shifting/)      | Alpha
+| [Circuit Break](https://istio.io/docs/tasks/traffic-management/circuit-breaking/)      | Alpha
+| [Mirroring](https://istio.io/docs/tasks/traffic-management/mirroring/)      | Alpha
+| [Ingress Traffic](https://istio.io/docs/tasks/traffic-management/ingress/)      | Alpha
+| [Egress Traffic](https://istio.io/docs/tasks/traffic-management/egress/)      | Alpha
+| [Egress TCP Traffic](https://istio.io/docs/tasks/traffic-management/egress-tcp/)      | Alpha
```
[the remark this answers is not in the document]
Comment (jasmine-jaksic) [reply content/about/feature-stages.md]: Updated the links

File: content/about/feature-stages.md (hunk 2 above)
Comment (jasmine-jaksic) [reply content/about/feature-stages.md]: You are right, changed installation to beta

File: content/about/feature-stages.md (hunk 1 above)
Comment (jasmine-jaksic) [reply content/about/feature-stages.md]: Removed..thanks for review/feedback

File: content/about/feature-stages.md (hunk 3 above)
Comment (jasmine-jaksic) [reply content/about/feature-stages.md]: Removed for now..will include after TCP and Websocket are fixed

**Pull request merged**